src/python_testing/DEMTestBase.py

```diff
@@ -0,0 +1,220 @@
+#
```

Comment:
@PeterC1965 - this file is renamed in the DEM TEST PR - to be TC_DEMTestBase.py
So I think you should remove this file from this PR now.

Reply:
I have removed this file.